pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -145,6 +143,17 @@ message CSVWriterSpec {
   optional roachpb.CSVOptions options = 3 [(gogoproto.nullable) = false];
   // chunk_rows is num rows to write per file. 0 = no limit.
   optional int64 chunk_rows = 4 [(gogoproto.nullable) = false];
+
+  // Compression list of the compression codecs which are currently
+  // supported for CSVWriter spec
+  enum Compression {
```

Comment:
I think "FileCompression" is enough of an independent concept we could move it top-level instead of nesting in CSV spec.

Reply:
sure